README.md

```diff
@@ -39,7 +39,7 @@ This enables customers to reuse our existing connector and components in their p
 
 ### Project structure
 
-This project is an Adobe I/O Application, to be deployed on [Adobe I/O Project Firefly](https://www.adobe.io/apis/experienceplatform/project-firefly.html).
+This project is an App Builder, to be deployed on [Adobe I/O](https://www.adobe.io/app-builder/).
```

Comment:
This project is an application to be deployed to Adobe Developer App Builder

Reply:
Changed

Reply:
Done